examples/fineweb_edu_classifier_example.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import os
+
+os.environ["DASK_DATAFRAME__QUERY_PLANNING"] = "False"
+import time
+
+import torch
+from crossfit import op
+from crossfit.backend.torch.hf.model import HFModel
+from transformers import AutoConfig, AutoModelForSequenceClassification
+
+from nemo_curator.datasets import DocumentDataset
+from nemo_curator.utils.distributed_utils import get_client
+from nemo_curator.utils.script_utils import ArgumentHelper
+
+EDU_CLASSIFIER_PATH = "HuggingFaceTB/fineweb-edu-classifier"
+
+
+class FinewebEduClassifier(HFModel):
```

Comment:
Do we want to integrate this into https://github.com/NVIDIA/NeMo-Curator/blob/main/nemo_curator/modules/distributed_data_classifier.py?

Reply:
Not right now, This just serves as an example on how to use it. Unsure if we want to combine it there ? 

@arhamm1 , Any opinions ?

Reply:
Yeah this is a tricky one. On one hand I think having an example of how people can extend our code to their own model is important. On the other, this model is super relevant and probably should be included in NeMo Curator. Perhaps we do both?

Reply:
We are good to have an example right now, based on interest we can also get it integrated